BlockSettleUILib/BSTerminalMainWindow.ui

```diff
@@ -758,6 +966,9 @@
    <container>1</container>
   </customwidget>
  </customwidgets>
- <resources/>
+ <resources>
+  <include location="armory.qrc"/>
+  <include location="armory.qrc"/>
```

Comment:
Duplication again

Reply:
Let's please do not touch this part, this is auto generated by qt creator, so looks like it the some bug. Editing this part in texteditor could bring much more trouble then having like this. If there any other way to update it via Qt creator, please specify, otherwise let's it, since it's not critical as for me. Thank you